src/visualizer/svg.js

```diff
@@ -15,66 +17,160 @@ const {
     const svgLineSpace = svgWidth * 0.04; // the amount of space between each line, by the width of the svg
     const svgLineWidth = svgWidth * 0.01;
     const svgTextSize = svgLineSpace * 0.5;
-    const RANGEWIDTH = 22;
+    const LSRANGEWIDTH = 22; // Learningstyle rangewith, from -11 to 11 = 22
+    const WAHRANGEWIDTH = 2; // Working At Home rangewith, from -1 to 1 = 2
```

Comment:
Inconsistent naming convention for const "global" variables

Reply:
const svgTextSize = svgLineSpace * 0.5;
    const learningStyleRangeWidth = 22; // Learningstyle rangewith, from -11 to 11 = 22
    const workingAtHhomeRangeWidth = 2; // Working At Home rangewith, from -1 to 1 = 2